Website/app/docs/Quickstart.md

```diff
@@ -0,0 +1,19 @@
+# A general overview of Dirfty
+
+Dirfty is an open source Interactive file downloader software built using Java and NextJS, Its backend is powered by Java and the front end consists of NextJS and TailwindCSS
+
+With the help of Dirfty, you can download files and videos from various sources such as YouTube, Instagram etc.. You can also download files by providing the link address if it's downloading way too slowly inside your browser.
+
+## Main Features of Drifty
+
+- Free and Open Source forever
+- Faster downloading of files, just like IDM 
+- Youtube, Instagram, Twitter/X etc.. Platforms supported
+- Both CLI (Command Line Interface) and GUI (Graphical User Interface) are available
+- Parallel downloading support provided
+
+# Let's get you running
+
+##  A simple demo on how to get started with the GUI of Drifty
```

Comment:
The Drifty CLI demo is not covered. Please consider adding it :grin:.

Reply:
But the CLI will be only covered for the windows machine @SaptarshiSarkar12 as I am running on windows that's why

Reply:
I'll provide you some resources and texts. Then, can you do it :grin:?